Makefile.defines

```diff
@@ -15,22 +15,10 @@
 #  limitations under the License.
 #*******************************************************************************
 
-#extract TARGET_NAME/TARGET_ID from the SDK to allow for makefile choices
-TARGET := nanos
-TARGET_ID:=$(shell cat $(BOLOS_SDK)/include/bolos_target.h | grep TARGET_ID | cut -f3 -d' ')
-TARGET_NAME:=$(shell cat $(BOLOS_SDK)/include/bolos_target.h | grep TARGET_ | grep -v TARGET_ID | cut -f2 -d' ')
-TARGET_VERSION:=$(shell cat $(BOLOS_SDK)/include/bolos_version.h | grep define | cut -f2 -d'"')
-SDK_NAME    := "nanos-secure-sdk"
-SDK_VERSION := $(shell git -C $(BOLOS_SDK) describe --tags --exact-match  --match "v[0-9]*" --dirty)
-SDK_HASH    := $(shell git -C $(BOLOS_SDK) describe --always --dirty --exclude '*' --abbrev=40)
-ifeq ($(SDK_VERSION),)
-SDK_VERSION := "None"
-endif
-ifeq ($(SDK_HASH),)
-SDK_HASH := "None"
-endif
+ifeq ($(__MAKEFILE_DEFINES__),)
```

Comment:
Add a comment to explain what this does and who sets it

Reply:
This is somehow an header guard, to allow multiple `include $(BOLOS_SDK)/Makefile.defines` without issues.
this has been merged in SDK master